### DISCUSSION TITLE
Onewindow new record 🕘

### DISCUSSION THREAD

**Discussion by BurtNotGoodAnything** [nzp-team/nzportable#1049] [Records and Speedruns]
![fte-20241219215800-0](https://github.com/user-attachments/assets/c0d4a948-93c8-4ac8-bbec-4861912675c6)

**Comment by zalopa** [comment]
![fte-20250707164747-0](https://github.com/user-attachments/assets/6eaf1e23-172e-4bbb-94a5-d93c1051d672)
![fte-20250707164750-0](https://github.com/user-attachments/assets/40c25a3b-9556-4732-872c-d82bcb629852)
![fte-20250707164827-0](https://github.com/user-attachments/assets/ddd4e7a3-d8a9-43c8-aea6-a39feefe0873)
new record!
